### DISCUSSION TITLE
无法为开启merge on write的unique模型的表的存量数据添加倒排索引

### DISCUSSION THREAD

**Discussion by yjx-hd** [apache/doris#25572] [A - General / Q&A]
为表添加倒排索引后，执行SHOW ALTER TABLE COLUMN;
显示事务id为-1
![image](https://github.com/apache/doris/assets/73015876/6ba8fcd0-bdcc-4e83-8ec8-500fa8d2b4b4)
为存量数据增加倒排索引，执行SHOW BUILD INDEX;发现任务一直在running，progress一直在0/16
![image](https://github.com/apache/doris/assets/73015876/b5e3e06f-9bfc-4b39-a9c0-ae92f0334a55)

**Comment by LemonLiTree** [comment]
which doris version？

**Reply by yjx-hd** [reply]
hi ,Doris's version is 2.0.1

**Reply by LemonLiTree** [reply]
Upgrade to 2.0.2
